docs/reference/schema-lang.md

```diff
@@ -254,6 +254,76 @@ It is **highly** recommended to put doc comments on all definitions, relations a
 
 <InlinePlayground reference="Vwput-WCrLaz"/>
 
+## Common Patterns
+
+### Group membership
+
+Apply specific users or members of a group to a permission on an object type.
+
+<details>
+    <summary>Example</summary>
+    <p>
+        In this example, a group can have users as admins and as members.
+        Both admins and members are considered to have membership in the group.
+        A role can be applied to individual users and groups.
+        All individually applied users as well as members for applied groups will have the `allowed` permission.
+    </p>
+
+<InlinePlayground reference="CNe3PXNuhypm"/>
+</details>
+
+### Super-admin / site-wide permissions
```

Comment:
"Global admin" -- let's avoid using characters like `-` and `/` in markdown headers so that URLs don't clash

Reply:
The headers are slug-ified but "Global admin" is more succinct